src/data/reducers/enterpriseList.test.js

```diff
@@ -0,0 +1,65 @@
+import enterpriseList from './enterpriseList';
+import {
+  FETCH_ENTERPRISE_LIST_REQUEST,
+  FETCH_ENTERPRISE_LIST_SUCCESS,
+  FETCH_ENTERPRISE_LIST_FAILURE,
+} from '../constants/enterpriseList';
+
+const initialState = {
+  enterprises: null,
+  loading: false,
+  error: null,
+};
+
+describe('enterpriseList reducer', () => {
+  it('has initial state', () => {
+    expect(enterpriseList(undefined, {})).toEqual(initialState);
+  });
+
+  it('updates fetch enterpriseList request state', () => {
```

Comment:
I found this wording confusing and changed it in the  authentication reducer here: https://github.com/edx/edx-portal/pull/22/files#diff-e7c432af33f88589df966bdc2261a45aR22

Up to you if you want to change it here